src/libexpr/primops.cc

```diff
@@ -792,12 +803,18 @@ static void prim_findFile(EvalState & state, const Pos & pos, Value * * args, Va
             % path % e.path % pos);
     }
 
-    mkPath(v, state.findFile(searchPath, path, pos).c_str());
+    v.setPath(state.findFile(searchPath, path, pos).c_str());
 }
 
+alignas(16) static const char sRegular[] = "regular";
```

Comment:
This string is because we give this string as argument of `setStringNoCopy`, which expects that string pointer to be properly aligned.  As this string is a static string, the pointer is not guaranteed to be aligned otherwise.